### DISCUSSION TITLE
Copilot Cancellation

### DISCUSSION THREAD

**Discussion by Crypto-stinger** [community/community#140682] [Copilot Conversations]
What a process to cancel Copilot! 5 steps which turn into 9 and I still didn't manage to do it. 1 cancellation button is all that is needed. Even Copilot can't do it, outside its scope it responded. Well it's outside mine also! Cancelled the payment. Much easier.

**Comment by Akash1134** [comment]
We now have the option to self-serve refunds for Copilot. Please head over to the Help with your GitHub account section and save yourself some time.

Note that, in most cases, the Virtual Agent may be able to offer an immediate refund. But, if your concern is not resolved, you can always use the contact form to [get in touch with our billing team](https://support.github.com/contact)).

Can you please try following these steps?:

1. Follow this specific link to our [contact form](https://support.github.com/contact/account?flow=refund_copilot_individual)
2. Scroll down a bit to see a blue button activated to "Try our virtual agent". 
3. Follow the prompts to request the cancellation and prorated refund.

This should automatically process the refund and cancellation and send a confirmation. If that does not work out please contact our [support team](https://support.github.com/contact).
